docs/first-model/lua-scripting.rst

```diff
@@ -1,33 +1,79 @@
 Lua scripting
 =============
 
-.. warning::
-
-   This page is under construction.
+Material and frictional parameters used in Tandem simulation are written in Lua script. Here's key parameters and their descriptions:
```

Comment:
Material and frictional parameters used in Tandem simulation**s** are written in Lua script. **Here are the** key parameters and their descriptions:

Reply:
overall, I would just copy paste the file to grammarly, and try to fix the typos.

Reply:
Sure, sorry for making you working on the grammar and typos. I will run them in the grammarly and fix all the typos.